tmd/view/view.py

```diff
@@ -318,17 +314,18 @@ def neuron(
     kwargs["new_fig"] = False
     kwargs["subplot"] = subplot
 
-    if not nosoma:
+    if plot_soma:
         soma(nrn.soma, plane=plane, hadd=hadd, vadd=vadd, **kwargs)
 
     h = []
     v = []
 
     to_plot = []
 
-    if rotation == "apical_dendrite":
+    if apical_alignment:
         angle = np.arctan2(nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1])
```

Comment:
I suspect this will fail if there is no apical tree. Could we raise if neuron has no apical?

Reply:
Indeed, thanks!
I rebased on master and fixed this.